integration/networking/port_mapping_linux_test.go

```diff
@@ -400,3 +400,43 @@ func TestAccessPublishedPortFromRemoteHost(t *testing.T) {
 		}
 	}
 }
+
+func TestRestartUserlandProxyUnder2MSL(t *testing.T) {
```

Comment:
The name of this test is a bit cryptic; what's the `2MSL` here?


Perhaps it'd be good to add a GoDoc to this test, so that it's clear what we're testing for (perhaps some inline comments as well; i.e. why do we 

- create a container, test for a 404
- remove it
- create a new container (without "using" it)

Reply:
MSL stands for Maximum Segment Lifetime, and 2x MSL, as per defined in RFC 793, is the delay before a TCP connection in TIME_WAIT state (ie. fully closed) is definitively considered as CLOSED.

I think this name does convey exactly what's being tested, but you need to be familiar with networking terms to get its meaning. Other names might get lengthy, eg. `TestRestartUserlandProxyWithTimeWaitConnections`.

I'll add a GoDoc and inline comments to explain what's being done and why.

Reply:
Ah, right! I wasn't familiar with that one, and wondered if `2MS` was 2 millisecond or `<whatever>` 😂